### DISCUSSION TITLE
Payee Manager/management under tree, not as popup

### DISCUSSION THREAD

**Discussion by altinturk** [moneymanagerex/moneymanagerex#6816] [Ideas]
Hey everyone.

I now come to realize i need a new payee manager - i'm having a long list of payees.
I was thinking of adding a "payees" item under the tree like budgets, accounts, etc where one click can list the payees and show some basic info. double click may open that specific payee and i can update its information on the popup.

now i need to scroll through the popup through all the payees, which is manageable, but a longer, better, non-popup list could work better there. Naturally that can be optional, if someone wants to use the old school popup list, that can still be used.
What do you think?

**Comment by n-stein** [comment]
What is the advantage of scrolling through a long list in the navigation pane versus scrolling through the exact same long list in the payee manager where you have the additional advantage of a filtering capability (via the search bar)? 

Seems you're saving at most one click -- two clicks (or three keystrokes) gets you into the payee manager which shows you all the basic info, and from there double clicking opens any payee to update its information.

**Reply by altinturk** [reply]
if i have many 'payees' to update every month i go to the expense, check the transaction, edit transaction, edit payee or create a new one from the popup. that's for each transaction i need to enhance. plus, i lose the position on the popup, so everytime this is rinse & repeat. 
I like the tabbed UI on gnucash where you can switch from one tab to another doing two things at once in parallel.
i imagined -instead of tabs for obv reasons- an entry there under the tree, and a screen where i can see payees and some details, so i can see and fill the unspecified ones quickly. 
additionally, imagine a screen like outlook contacts where above the payee info, and below a list of transactions related to that payee. a reverse query on the db. 
i should check and understand the code to say if any of those are feasible though. anything above sounds interesting?

**Comment by georgeef** [comment]
I would prefer to see all tables (Account, Payee, Category, Currency, Tag) as a panel (in the big main area of the main window), containing all information, instead of a popup. The Payee popup (unlike other popups) contains all information, but I still prefer a panel for a complete table. A popup feels better for limited and more focused information.

However, I wouldn't prefer to see detailed lists of items inside the Navigator tree. This feature would make the Navigator too complex, and I doubt that it would accelerate the processing of individual items. The Navigator tree already contains individual accounts, but for a different purpose: they give quick access to relevant transactions, not to the accounts themselves.

**Reply by altinturk** [reply]
This actually is a better idea for `Tools > Organize * ` menu items to be enhanced. The usual popup, instead, remains as usual on transactions where a change is done each time per-transaction. That's a better approach imo, not overcrowding the menu, but still giving a better payee/category/currency/tag management. nice comment.

**Reply by nmsleal** [reply]
Completely agreed!
